src/pkg/control/options.go

```diff
@@ -13,6 +13,8 @@ type Options struct {
 	SkipReduce         bool            `json:"skipReduce"`
 	ToggleFeatures     Toggles         `json:"toggleFeatures"`
 	Parallelism        Parallelism     `json:"parallelism"`
+	User               string          `json:"user"`
+	Host               string          `json:"host"`
```

Comment:
I'm fine with adding `Repo` to it. I'd prefer to stay away from mentioning kopia directly in places and/or exporting anything kopia related to the SDK level because really the fact that corso is built on kopia is an implementation detail